.travis.yml

```diff
@@ -9,6 +9,7 @@ go:
   - tip
 
 install:
+  - go get -u golang.org/x/vgo
```

Comment:
Why do you need this with Go 1.11?

Reply:
Thanks for that -- looking at https://dave.cheney.net/2018/07/16/using-go-modules-with-travis-ci now

Reply:
https://roberto.selbach.ca/intro-to-go-modules/ is the best user-guide I have found.

I converted one of our projects to use Go 1.11 with travis. See here: https://github.com/appscode/go/commit/97214a81824c6834bc34a0a0d5f1e17ed7099ef9